style/style.css

```
@@ -0,0 +1,168 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  margin-top: 2%;
+  margin-bottom: 2%;
+  margin-right: 3%;
+  margin-left: 3%;
+  background-color #F8F8F8;
+
+}
+
+ul {
+  list-style: none;
+}
+
+h1 {
+  font-size: 200%;
+}
+
+h2 {
+  text-align: center;
+}
+
+h3 {
+  text-align: left;
+  padding-left: 40px;
+}
+
+.nav ul {
+  display: grid;
+  grid-template-columns: 60% 30% 20% 10% 30%;
+  grid-template-rows: 20%;
+  font-weight: bold;
+}
+
+.section1 {
+  background-color: #A9A9A9;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  text-transform: capitalize;
+  border-radius: 10px;
+  padding: 5px;
+}
+
+.section1 sub {
+  padding: 5px;
+}
+
+.section1 a {
+  border: 1px solid #A9A9A9;
+  text-decoration: none;
+  padding: 10px;
+  border-radius: 10px;
+  color: #363535;
+}
+
+.section2 {
+  background-color: #A9A9A9;
+  text-align: center;
+  font-style: italic;
+  font-size: 100%;
+  border-radius: 10px;
+}
+
+.section2 p {
+  text-align: right;
+  padding: 5px;
+}
+
+.section2 p::before {
+  content: '--';
+}
+
+blockquote::before, blockquote::after {
+  content: '"';
+}
+
+.section3 {
+  background-color:#A9A9A9;
+  border-radius: 10px;
+}
+
+.section3 ul {
+  display: flex;
+  flex-direction: row;
+  padding: 5px;
+}
+
+.section3 ul li p {
```

```suggestion
.section3 ul li p {
```
I noticed that you created an id for the li's but didn't use it and I was thinking you could use that here if you wanted to by doing .section3 #confortable p  or you don't need it at all.